pkg/i2gw/notifications/config.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package notifications
+
+import (
+	"os"
+
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/jedib0t/go-pretty/v6/text"
+)
+
+const (
+	bgColor = text.BgBlack
+
+	infoColor    = text.FgHiWhite
+	warningColor = text.FgHiMagenta
+	errorColor   = text.FgRed
+	defaultColor = text.FgWhite
+)
+
+func newTableConfig() table.Writer {
+	t := table.NewWriter()
+
+	t.SetOutputMirror(os.Stdout)
+	t.SetRowPainter(func(row table.Row) text.Colors {
+		switch row[1] {
```

Comment:
```suggestion
		switch messageType := row[1]; messageType {
```

I know this is 100% the same, but I feel it'd increase code readability

Reply:
Also, I think we should perform some checks on the row length. To make it bulletproof, we need to be sure that `len(row) > 1`

Reply:
That is automatically handled by the `table` package. If there are no rows, then it wont render anything. So the check is unnecessary